.github/workflows/backup-firestore.yml

```diff
@@ -0,0 +1,21 @@
+name: Backup Firestore
+
+on:
+  schedule:
+    - cron:  '5 4 11 1,7 *' # 4:05am on 1/11 and 7/11
```

Comment:
Why not 4:00am on the 1st or 15th?

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

> Note: The schedule event can be delayed during periods of high loads of GitHub Actions workflow runs. High load times include the start of every hour. To decrease the chance of delay, schedule your workflow to run at a different time of the hour.